baseledger/x/bridge/types/params.go

```diff
@@ -9,7 +14,10 @@ var _ paramtypes.ParamSet = (*Params)(nil)
 
 // ParamKeyTable the param key table for launch module
 func ParamKeyTable() paramtypes.KeyTable {
-	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{
```

Comment:
just curious, what is this initialization here and why it is empty for both params?